applications/frontend/src/components/header/Banner.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+
+import { UserInfo } from '../../types/user';
+
+const useStyles = makeStyles((theme) => ({
+  mainFeaturedPost: {
+    position: 'relative',
+    backgroundColor: theme.palette.grey[800],
+    color: theme.palette.common.white,
+    margin: theme.spacing(1),
+    backgroundImage: 'url(images/banner.png)',
+    backgroundSize: 'cover',
+    backgroundRepeat: 'no-repeat',
+    backgroundPosition: 'center',
+  },
+  overlay: {
+    position: 'absolute',
+    top: 0,
+    bottom: 0,
+    right: 0,
+    left: 0,
+    backgroundColor: 'rgba(0,0,0,.3)',
+  },
+  mainFeaturedPostContent: {
+    position: 'relative',
+    padding: theme.spacing(3),
+    [theme.breakpoints.up('md')]: {
+      padding: theme.spacing(6),
+      paddingRight: 0,
+    },
+  },
+  button: {
+    borderRadius: "5em",
```

Comment:
We may set the global border radius from the theme to get consistency

Reply:
these are the "special" rounded buttons

Reply:
this is a special banner button so I leave it here